draft-ietf-ppm-dap.md

```diff
@@ -1380,7 +1380,10 @@ This message consists of:
 Finally, the Leader sends a PUT request to
 `{helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}`. The payload
 is set to `AggregationJobInitReq` and the media type is set to
-"application/dap-aggregation-job-init-req".
+"application/dap-aggregation-job-init-req". After receiving HTTP status 202
+Created, the Leader proceeds to poll GET requests to the aggregation job URI
+until receiving HTTP status 200 OK with a body containing an
+`AggregationJobResp`.
```

Comment:
This new text should be moved a few lines below, to where it says "The Helper's response will be...".

First, it's awkward to discuss response handling and then have a sentence discussing Leader->Helper authentication, which is relevant to the request.

Second, the paragraph beginning "The Helper's response will be an `AggregationJobResp` message..." should be updated, because it's not clear whether that's talking about the PUT or a subsequent GET. 

Second,